.github/workflows/qa-symlink.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: "Check for missing .qa links"
+on: [pull_request]
+
+jobs:
+  pull_request:
+    name: "Check for missing .qa links"
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout ceph.git
+        uses: actions/checkout@v4
```

Comment:
We should pin official GH actions too. @djgalloway was suggesting (and I agree) that we consider all version tags as vulnerable, no matter who's the author, and we use SHA-1 instead.

Reply:
Do we have an open tracker for that yet? I went ahead and merged this as we can fix that later.